packages/nextjs/app/builders/0xaC1bBf2Aba61D6fC6fCbade72eCe3362de4724c3/page.tsx

```diff
@@ -0,0 +1,54 @@
+import Image from "next/image";
+import type { NextPage } from "next";
+
+// Using Next.js optimized Image component
+
+const AVATAR_URL = "https://avatars.githubusercontent.com/u/583231?v=4";
+const WALLET_ADDRESS = "0xaC1bBf2Aba61D6fC6fCbade72eCe3362de4724c3";
+
+const SOCIALS = [
+  { name: "Twitter", url: "https://twitter.com/yourhandle", icon: "🐦" },
+  { name: "GitHub", url: "https://github.com/yourhandle", icon: "💻" },
+  { name: "LinkedIn", url: "https://linkedin.com/in/yourhandle", icon: "🔗" },
+];
+
+const Web3ProfilePage: NextPage = () => {
+  return (
+    <main className="min-h-screen bg-gradient-to-br from-indigo-100 to-teal-50 flex items-center justify-center font-sans">
+      <section className="bg-white rounded-xl shadow-xl p-10 max-w-md w-full text-center">
+        <Image
+          src={AVATAR_URL}
+          alt="Avatar"
+          width={100}
+          height={100}
+          className="rounded-full object-cover shadow-sm mx-auto mb-4"
+          priority
+        />
```

Comment:
There’s an error related to this image.

You can rebase your branch onto `main` to get the Next.js remotes config.